sql/core/src/test/scala/org/apache/spark/sql/BitmapExpressionsQuerySuite.scala

```diff
@@ -217,7 +217,7 @@ class BitmapExpressionsQuerySuite extends QueryTest with SharedSparkSession {
       errorClass = "DATATYPE_MISMATCH.UNEXPECTED_INPUT_TYPE",
       parameters = Map(
         "sqlExpr" -> "\"bitmap_count(a)\"",
-        "paramIndex" -> "0",
+        "paramIndex" -> "1",
```

Comment:
Should be `first`?

Reply:
Yea, `two` were `missed` and have been fixed.